[TECH-1831] Improving tests + cleanup

av-alexistoledo opened the pull request.

av-alexistoledo committed:
faq screen

av-alexistoledo committed:
contact from conference

av-alexistoledo committed:
theme fallback

av-alexistoledo committed:
fix zindex

av-alexistoledo committed:
name update

av-alexistoledo committed:
styling

av-alexistoledo committed:
enabling playwright to test in mobile viewports

av-alexistoledo committed:
prettier

av-alexistoledo committed:
adds unit test for trackedballotmanager component

av-alexistoledo committed:
linting

av-alexistoledo committed:
avoiding double logo

av-alexistoledo committed:
refactor and test for contentlayout

av-alexistoledo committed:
adding autofocus

av-alexistoledo committed:
removing fontawesome

av-alexistoledo committed:
styling fix

av-alexistoledo committed:
removing comments

av-alexistoledo committed:
unifying routes

av-alexistoledo committed:
fixing padding locale selector

av-emil commented on client/src/router/index.ts:
```diff
@@ -36,7 +36,7 @@ const router = createRouter({
     {
       name: "BallotVerificationLanding",
-      path: "/:locale/:electionSlug/verification",
+      path: "/:locale/:electionSlug/verify",
```
Consider adding a redirect on the old URL, in case this has already been deployed to production, and someone has stored their old links.

av-alexistoledo replied:
Verification was a new route I added so no one is using it so far. I just decided to use the same one we were already using instead of having two different paths for the same flow.

av-alexistoledo committed:
more style fixing

av-alexistoledo committed:
moar styles

av-alexistoledo committed:
[TECH-1876] Fallback styles (#32)

Introduces fallback styles in order to be used when there is no endpoint
set.

av-lukas committed:
Merge branch 'TECH-1840-contact-us' into TECH-1831-tests

av-alexistoledo committed:
[TECH-1840] Contact us (#31)

The contact menu in the header comes now from Conference. If it is not
set or we're not using Conference it won't be shown.

Relies on https://github.com/aion-dk/conference/pull/601

av-lukas committed:
Merge branch 'TECH-1839-implement-changes-on-faq-screen' into TECH-1831-tests

av-lukas committed:
Merge branch 'TECH-1838-implement-changes-on-activity-logs-screen' into TECH-1831-tests
